content/docs/command-reference/exp/branch.md

```diff
@@ -15,26 +15,28 @@ positional arguments:
 
 ## Description
 
-Makes a named Git
-[`branch`](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging)
-containing the target `experiment` (making it
-[persistent](/doc/user-guide/experiment-management#persistent-experiments)). For
```

Comment:
So, the previous version that says "checkpoints -> multiple commits" was not correct?

Reply:
That sentence about checkpoints->many commits is unchanged. Well, was unchanged until just now that I rephrased this paragraph but the idea is still there (will comment separately, resolving here).